### DISCUSSION TITLE
Review Bonsai.ONIX naming conventions

### DISCUSSION THREAD

**Discussion by glopesdev** [open-ephys/bonsai-onix1#2] [Ideas]
It would be useful for us and users if we could align naming conventions of Bonsai.ONIX to match general C# and Bonsai package standards. Some ideas below:

- [x] Discuss whether `Bonsai` is the best namespace for the package
- [x] Review casing conventions for classes and enums

The above points can be split off into separate discussions later and new ones added after further review.

**Comment by jonnew** [comment]
I have no opinion on Casing conventions other than maintainability and fitting with default linter options. So I guess I do have an opinion.
